addons/mrp/models/mrp_unbuild.py

```diff
@@ -13,10 +13,20 @@ class MrpUnbuild(models.Model):
     _order = 'id desc'
 
     def _get_default_location_id(self):
-        return self.env.ref('stock.stock_location_stock', raise_if_not_found=False)
+        stock_location = self.env.ref('stock.stock_location_stock', raise_if_not_found=False)
+        try:
+            stock_location.check_access_rule('read')
```

Comment:
The problem with this is that if `stock.stock_location_stock` has been deleted, `stock_location` will be `None`, and that will crash.

You could try:
```python
if stock_location and stock_location._filter_access_rules('read'):
    return stock_location.id
else:
    return self.env['stock.warehouse'].search([('company_id', '=', self.env.user.company_id.id)], limit=1).lot_stock_id.id
```

This way, you can get rid of the `try/except`